reactive/k8s.py

```diff
@@ -7,18 +7,20 @@
 from charms.reactive import when
 from charms.reactive import when_not
 
+from charmhelpers.core import hookenv
 from charmhelpers.core.hookenv import status_set
 from charmhelpers.core.templating import render
 
 from os import getenv
 from contextlib import contextmanager
 
+
 @when('docker.available')
 @when_not('etcd.available')
 def relation_message():
     '''Take over messaging to let the user know they are pending a relationship
     to the ETCD cluster before going any further. '''
-    status_set('waiting', 'Relate me to ETCD')
+    status_set('waiting', 'Waiting for relation to ETCD')
```

Comment:
Would this technically be blocked instead of waiting? this was my bad when i wrote the status update.